plugins/builds/triggers/helpers.js

```diff
@@ -686,8 +686,8 @@ function getSubsequentJobs(workflowGraph, startNode) {
     const nodeToEdgeDestsMap = Object.fromEntries(nodes.map(node => [node.name, []]));
 
     let start = trimJobName(startNode);
-    // In rare cases, WorkflowGraph and startNode may have different start tildes
 
+    // In rare cases, WorkflowGraph and startNode may have different start tildes
```

Comment:
Thanks for finding the error.
As other members have said, it looks like a bug to have job names in `name`, `src`, `dest` with `~` prefixes other than `~commit` and `~pr`.
(I have actually encountered such a case during my last PR, as this comment.)

Also, I think it is a bug that there is a`src` or `dest` with a name that is not in the `nodes` like `certify-db`, apart from the presence or absence of `~` prefix.